plover/oslayer/linux/keyboardcontrol_uinput.py

```diff
@@ -256,13 +276,7 @@
     },
 }
 
-us_qwerty = {
-    **LAYOUTS[DEFAULT_LAYOUT],
-    ";": e.KEY_SEMICOLON,
-    "'": e.KEY_APOSTROPHE,
-    "[": e.KEY_LEFTBRACE,
-}
-KEYCODE_TO_KEY = dict(zip(us_qwerty.values(), us_qwerty.keys()))
+KEYCODE_TO_KEY = dict(zip(LAYOUTS[DEFAULT_LAYOUT].values(), LAYOUTS[DEFAULT_LAYOUT].keys()))
```

Comment:
actually this doesn't even make sense, it should have just been a set and named `KEYCODES_TO_SUPPRESS` or something. But the original code is like that